int8 led_id
---
string result
string fail_reason 
